hoot-core-test/src/test/cpp/hoot/core/algorithms/changeset/ChangesetDeriverTest.cpp

```diff
@@ -39,42 +39,48 @@ namespace hoot
 
 class ChangesetDeriverTest : public HootTestFixture
 {
-    CPPUNIT_TEST_SUITE(ChangesetDeriverTest);
-    CPPUNIT_TEST(runTest);
-    CPPUNIT_TEST_SUITE_END();
+  CPPUNIT_TEST_SUITE(ChangesetDeriverTest);
+  CPPUNIT_TEST(runTest);
```

Comment:
A couple of these tests look like I changed more than I did.  Here I just updated the spacing from 4 to 2.